recipes/aneris/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.2.0" %}
+{% set sha256 = "624412e196fcefc2708d472c6840509492f91327e1fabcaf46fe61ab9a0705ff" %}
+
+package:
+  name: aneris
+  version: {{ version }}
+
+source:
+  url: https://github.com/iiasa/aneris/archive/v{{ version }}.tar.gz
```

Comment:
do you want to do this from GitHub rather than pypi? I did it in pypi here (https://github.com/conda-forge/staged-recipes/pull/6830/files) but I can't remember why exactly (provenance perhaps).